### PR TITLE
ref(slack): increase max chars for user feedback issues

### LINKED ISSUE #68433: Don't trim the message char count for User Feedback Slack Alerts
Feedback from a customer on the Slack discussions:

>Hello! We are happy with the new notification format, except one small detail: these notifications are now being **trimmed to 254 symbols**.
>
>It is not convenient for us because not all developers nor support agents, for security reasons, have access to Sentry. Before this change, they were able to read the whole message in Slack.
>
>Is there any chance of keeping the whole User Feedback message when sending it to Slack? Thanks.

_Originally posted by @dmitry-textmagic in https://github.com/getsentry/sentry/discussions/61601#discussioncomment-9041708_

### PR DESCRIPTION
**Pull request by cathteng**
Resolves https://github.com/getsentry/sentry/issues/68433

tl;dr a customer wants longer (preferably the entire) text lengths for User Feedback issues

### CODE REVIEW

File: tests/sentry/integrations/slack/test_message_builder.py [hunk 1]
```diff
@@ -978,9 +977,44 @@ def test_build_performance_issue_block_kit(self):
 
     @with_feature("organizations:slack-block-kit")
     def test_block_kit_truncates_long_query(self):
-        text = "a" * 5000
-        block = BlockSlackMessageBuilder().get_markdown_quote_block(text)
-        assert "a" * 253 + "..." in block["text"]["text"]
+        # text = "a" * 5000
+        # block = BlockSlackMessageBuilder().get_markdown_quote_block(text)
+        # assert "a" * 253 + "..." in block["text"]["text"]
+
```
Comment (cathteng) [inline tests/sentry/integrations/slack/test_message_builder.py:983]:
```suggestion
```

**Review by iamrajjoshi — APPROVED** [review]
🚀  qq - why can't everyone have the larger block text length?

**Comment by cathteng** [comment]
@iamrajjoshi lots of text increases the screen real estate of each message, which irritates some users. but we can't please everybody. it seems to be ok for user feedback because the entire content of the issue is the text. for this particular issue, the customer has contractors and other employees that don't have access to the sentry instance and they need to be able to read the user feedback without going to sentry
